src/module/vue/components/site/site-denizenbox.vue

```diff
@@ -7,26 +7,24 @@
   >
     <label
       class="nogrow"
-      style=" flex-basis: 4em; line-height: 26px;white-space: nowrap"
+      style="flex-basis: 4em; line-height: 26px; white-space: nowrap"
     >
-      <span v-if="denizen.low === denizen.high">{{ denizen.low }}</span>
-      <span v-else>{{ denizen.low }}–{{ denizen.high }}</span>
+      <span v-if="denizen.range[0] === denizen.range[1]">{{
+        denizen.range[0]
+      }}</span>
+      <span v-else>{{ denizen.range[0] }}–{{ denizen.range[1] }}</span>
```

Comment:
I don't like this notation as much, but you're right, it's more consistent with Foundry internals. Le sigh.